Sources/Sentry/Public/SentryHttpInterceptor.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A class used to automatically track http requests.
+ * You can used it by passing it in you URL Session configuration manually
+ * or by turning on automatic instrumentation.
+ */
+@interface SentryHttpInterceptor : NSURLProtocol
```

Comment:
having such a generic name, does it work for any sort of HTTP requests? using a 3rd party lib or not?

Reply:
likely yes, let's test and see if it works with the most famous HTTP client for iOS, https://github.com/Alamofire/Alamofire